doc/BBCode.md

```diff
@@ -61,17 +61,17 @@ table.bbcodes > * > tr > th {
   <td><span style="color:  red;">red</span></td>
 </tr>
 <tr>
-  <td>[url=http://www.friendica.com]Friendica[/url]</td>
-  <td><a href="http://www.friendica.com" target="external-link">Friendica</a></td>
+  <td>[url=http://friendi.ca]Friendica[/url]</td>
+  <td><a href="http://friendi.ca" target="external-link">Friendica</a></td>
 </tr>
 <tr>
-  <td>[img]http://friendica.com/sites/default/files/friendika-32.png[/img]</td>
-  <td><img src="http://friendica.com/sites/default/files/friendika-32.png" alt="Immagine/foto"></td>
+  <td>[img]https://raw.githubusercontent.com/friendica/friendica/develop/images/friendica-32.jpg[/img]</td>
```

Comment:
Probably just the default branch you were viewing, but I'd rather use the `master` version just in case:

`https://raw.githubusercontent.com/friendica/friendica/master/images/friendica-32.jpg`